cli_tools/common/distro/distro.go

```diff
@@ -167,6 +206,44 @@ func newCommonLinuxRelease(distro string, major, minor int) (Release, error) {
 	}, nil
 }
 
+// windowsRelease uses marketing versions rather than NT versions.
+// Currently the only minor version is "r2". For example, the versions
+// 2012 and 2012r2 and *not* import compatible, and have different
+// minor versions. In the first case, the minor version is empty.
+// In the second it is "r2".
+type windowsRelease struct {
+	major, minor, architecture string
+}
+
+func (w windowsRelease) ImportCompatible(other Release) bool {
+	actualOther, ok := other.(windowsRelease)
+	compat := ok &&
```

Comment:
compat->compatible. Not much longer and others won't read it as `combat` :)

Reply:
done!